srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableSummary.java

```diff
@@ -78,18 +100,23 @@ public void visitMultiSelect(final MultiSelectVariable variable)
         {
             fTypeName = (variable.getDisplayType() == DisplayType.Radio) ?
                     "Radio Button" : "Drop-Down";
+            fOptionString = String.format("[%s]", Joiner.on(", ").join(variable.getOptions()));
         }
         
         @Override
         public void visitDiscreteNumerical(final DiscreteNumericalVariable variable)
         {
             fTypeName = "Discrete Numerical";
+            fOptionString = String.format("Range: %s Options: [%s]", 
+                    variable.getValidRange().toString(),
+                    Joiner.on(", ").join(variable.getOptions()));
         }
 
         @Override
         public void visitProcedure(final ProcedureVariable variable)
         {
             fTypeName = "Procedure Selection";
+            fOptionString = variable.getDisplayName();
```

Comment:
Since the only Procedure variable is called "Procedure", this ends up pretty redundant on the UI.